patches/@libp2p+interop+9.0.0.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/node_modules/@libp2p/interop/src/dht/peer-routing.ts b/node_modules/@libp2p/interop/src/dht/peer-routing.ts
```

Comment:
See the other comment about `Peer` vs `PeerInfo`, the DHT `.findPeer` method returns a `PeerInfo` which doesn't have the peer id appended so this patch shouldn't be necessary.